business/organization.ts

```diff
@@ -126,7 +126,7 @@ export class Organization {
     if (operationsIsCapable<IOperationsUrls>(operations, CoreCapability.Urls)) {
       this._baseUrl = `${operations.baseUrl}${this.name}/`;
       this._nativeUrl = `${operations.nativeUrl}${this.name}/`;
-      this._nativeManagementUrl = `${operations.nativeManagementUrl}organizations/${this.name}/`;
+      this._nativeManagementUrl = `${operations.nativeManagementUrl}${this.name}/`;
```

Comment:
Thanks... I need to fix this in a slightly different way but will work to get this one accepted.

(We have a React front-end that has a different URL structure!)